lib/model/card/action/scry_action.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:treasure_of_the_high_seas/model/game_state.dart';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+import 'card_action.dart';
+
+class ScryAction extends CardAction {
+  List<Resource> cost;
```

Comment:
This property is redundant as it's on the base class

Reply:
True. Fixed.